internal/app/referenceserver/checks.go

```diff
@@ -212,13 +212,14 @@ func checkCompression(expected v1.Compression, req *http.Request, feedback *feed
 		contentType := req.Header.Get("content-type")
 		var encodingHeader string
 		switch {
-		case contentType == grpcContentType || contentType == grpcWebContentType ||
-			strings.HasPrefix(contentType, grpcContentTypePrefix) ||
-			strings.HasPrefix(contentType, grpcWebContentTypePrefix):
+		case contentType == grpcContentType ||
+			strings.HasPrefix(contentType, grpcContentTypePrefix):
 			encodingHeader = "grpc-encoding"
 		case strings.HasPrefix(contentType, connectStreamContentTypePrefix):
 			encodingHeader = "connect-content-encoding"
-		case strings.HasPrefix(contentType, connectUnaryContentTypePrefix):
+		case contentType == grpcWebContentType ||
```

Comment:
gRPC-web uses `Content-Encoding` and ignores `Grpc-Encoding`.  See [docs](https://github.com/grpc/grpc-web/blob/master/doc/browser-features.md#compression).